.github/workflows/ci.yml

```diff
@@ -43,12 +43,12 @@ jobs:
 
       - name: Get tinygo runtime
         run: |
-          wget -P /tmp/ https://github.com/tinygo-org/tinygo/releases/download/v0.20.0/tinygo_0.20.0_amd64.deb
-          sudo dpkg -i /tmp/tinygo_0.20.0_amd64.deb
+          wget -P /tmp/ https://github.com/tinygo-org/tinygo/releases/download/v0.22.0/tinygo_0.22.0_amd64.deb
+          sudo dpkg -i /tmp/tinygo_0.22.0_amd64.deb
 
       - name: Build WASM
         run: |
-          tinygo build -o ./lib/compiler/astro.wasm -target wasm ./cmd/astro-wasm/astro-wasm.go
+          tinygo build -gc leaking -scheduler asyncify -no-debug -o ./lib/compiler/astro.wasm -target wasm ./cmd/astro-wasm/astro-wasm.go
```

Comment:
As I mentioned, `-gc leaking` is necessary to avoid over-eager garbage collection that was cleaning up functions before we had an opportunity to call them.

This is because we're spawning goroutines and invoking a `Promise.resolve` callback inside of it. If we don't stop garbage collection from happening, `resolve` is undefined by the time we try to invoke it.